File: Assignmentno21/task_3.java
import java.util.*;
class Logic 
{
    void CountFactors(int iValue)
    {
        int iCnt = 0;
        int iCount = 0;
        for(iCnt=1;iCnt<=iValue/2;iCnt++)
        {
            if((iValue%iCnt)==0)
            {
                System.out.print(iCnt);
                iCount++;
            }
        }
        System.out.print("The Factors will be:"+iCount);
    }

}
class task_3
{
    public static void main(String A[])
    {
        int iNo=0;
        Scanner sobj = new Scanner(System.in);
        Logic lobj = new Logic();

        System.out.println("Enter the number:");
        iNo = sobj.nextInt();
        lobj.CountFactors(iNo);


    }
}

File: Assignmentno-3/task_3.c
#include<stdio.h>
void PrintEven(int iNo)
{
    int iCnt = 0;
    if(iNo < 0)
    {
        iNo = -iNo;
    }
    for(iCnt = 1;iCnt<iNo;iCnt++)
    {
        if((iNo%iCnt) == 0 && (iCnt%2)==0)
        {
            printf("%d\t",iCnt);
        }
    }

}

int main()
{

    int iValue = 0;
    printf("Enter they number\n");
    scanf("%d",&iValue);

    PrintEven(iValue);

    return 0;
}